src/Elmish/React/Ref.purs

```diff
@@ -0,0 +1,24 @@
+module Elmish.React.Ref
+  ( Ref
+  , callbackRef
+  )
+  where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Elmish.Dispatch ((<?|))
+import Elmish.Foreign (class CanPassToJavaScript)
+import Unsafe.Coerce (unsafeCoerce)
+
+data Ref (el :: Type)
+
+instance CanPassToJavaScript (Ref a)
+
+callbackRef :: forall el. Maybe el -> (el -> Effect Unit) -> Ref el
+callbackRef ref setRef = unsafeCoerce $ setRef <?| \r -> case eqByReference r <$> ref of
```

Comment:
I generally think its helpful to document the coercion, (and ensure that it doesn't change out from under you) by writing a helper that is specialized. 
So something like: 
```
mkCallbackRef :: forall a. EffectFn1 a Unit -> Ref a
mkCallbackRef = unsafeCoerce
```
And then use `mkCallbackRef` at the callsite.

Reply:
Yeah good idea, I’ll do that